spec/views/posts/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "posts/edit", type: :view do
+  before(:each) do
+    @post = assign(:post, Post.create!(
+      :user_id => 1,
+      :content => "MyString"
+    ))
+  end
+
+  it "renders the edit post form" do
+    render
+
+    assert_select "form[action=?][method=?]", post_path(@post), "post" do
+
+      assert_select "input#post_user_id[name=?]", "post[user_id]"
+
+      assert_select "input#post_content[name=?]", "post[content]"
+    end
```

Comment:
I would actually get rid of the the blank lines on 15 and 17. I don't think the blank lines help with distinction here.